bibcheck/plugins/kekmove.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+##
+## This file is part of Invenio.
+## Copyright (C) 2015, 2016 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+""" Bibcheck plugin to take URLs with KEK identifiers from 8564_u marked
+    with 8564_y:KEKSCAN, store the identifiers in 035__a with 035__9:KEKSCAN
+    and delete the corresponding tag 856
+"""
+
+import re
+
+from invenio.bibrecord import record_add_field
+
+provenance = 'KEKSCAN'
+
+kekidre = re.compile(r'^https?//www-lib.kek.jp/cgi-bin/img_index\?(\d+)$')
```

Comment:
KEK ids should be either 7 or 9 digits depending on whether the year is 2 or 4 digits. The regexp should be more specific. I find 2 kek identifiers in 8564_u which are 8 digits -- and those won't result in resolvable urls